### ISSUE TITLE
Running on Raspberry Pi ?

### ISSUE THREAD

**Issue by dtbaker** [gnebehay/CppMT#38]
Hello,

Sorry I have OpenTLD running and interested in swapping over to CMT, just trying to get this running on Raspberry Pi and wondering if anyone has had success?

Thank heaps,
Dave

**Comment by abishekh** [comment]
Which RaspberryPi, I have it compiled for Jessie on a DragonBoard - could benefit from some GPU support - frame rate is too low.

**Comment by dtbaker** [comment]
Hey yep I got it compiled on Rasbian running on the Raspberry Pi3. Both the python and C++ versions ran quite nicely ( 7-10 FPS ) when inside with nice clean walls. However running it outside was horrible, way too many keypoints(?) in the high contrast surrounding. Only managed 1-2 FPS.

This one is running the Python CMT: https://www.youtube.com/watch?v=hvzohGuw8XI 

This one is running the C++ CMT: https://www.youtube.com/watch?v=sGqsEOVGPbE I swapped the opencv capture method over to <a href="https://github.com/cedricve/raspicam">raspicam</a> and noticed a speed increase. 

I don't have a video of it working outside because well it didn't work. I'm going to swap out CMT for glyph or color tracking. Then push the code to github and hopefully someone smarter than me can make some object tracking suggestions (still new to opencv).

Cheers.
